Cookbook/Technical-Documents/NewHiresCheckList.md

```diff
@@ -42,53 +42,32 @@ As an iOS Engineer, you should be in the following Slack channels:
 
 Make sure you also join your Tribe/Squad's Slack channels.
 
-## Setup your development environment
+## Install prerequisites
 
-Here's how to get the iOS project up and running.
+  - Xcode + its command-line tools
```

Comment:
```suggestion
  - Xcode 10.1 + it's command-line tools
```

Reply:
I think the original was correct.

Reply:
I agree. First `its` should be without an apostrophe, regardless. Also, I don't think we should mention a specific Xcode version number. The version number issue is only a temporary one when a new Xcode version is released.

What *might* be a good thing to mention is to install Xcode from the Apple Developer Downloads page and **not** from the Mac App Store, to avoid unexpected automatic updates.

Reply:
@mluisbrown Agreed all around. I hit merge just as you were posting this; I'll open a PR to add that note.